soc/arm/aspeed/ast10x0/Kconfig.series

```diff
@@ -12,5 +12,6 @@ config SOC_SERIES_AST10X0
 	select SOC_FAMILY_ASPEED
 	select CACHE
 	select CACHE_MANAGEMENT
+	select CACHE_ASPEED
```

Comment:
are you still missing `CONFIG_HAS_EXTERNAL_CACHE` somewhere though?

Reply:
I added in Kconfig.defconfg.series.  Should I move it to somewhere?
https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/arm/aspeed/ast10x0/Kconfig.defconfig.series#L18-L19

Reply:
interesting. I would have expected that change to be in this PR. Anyway, that's fine.